sktime/transformations/series/summarize.py

```diff
@@ -427,7 +427,7 @@ def _window_feature(Z, summarizer=None, window=None, bfill=False):
     Apply summarizer passed over a certain window
     of past observations, e.g. the mean of a window of length 7 days, lagged by 14 days.
 
-    Z: pandas Dataframe with a single column.
+    Z: pandas Dataframe with a single column or a pandas Series.
```

Comment:
Are you sure that this can be a pandas.Series object? I think it could be a SeriesGroupedByObject. 

Since it is not clear what this really is and this is no public method, I would propose to remove the changes to the docstring.

Reply:
Ye I have not really tested for pandas Series as such I will remove it.